scripts/build.py

```diff
@@ -770,37 +792,45 @@ def _findfiles(directory, exts):
             paths = _findlibs(libdir)
             self._add_files_to_zip(zip, paths, self.dynamo_home, topfolder)
 
-            # Android Jars (Dynamo)
-            jardir = os.path.join(self.dynamo_home, 'share/java')
-            paths = _findjars(jardir, ('android.jar', 'dlib.jar', 'r.jar'))
-            self._add_files_to_zip(zip, paths, self.dynamo_home, topfolder)
+            if platform in ['armv7-android', 'arm64-android']:
```

Comment:
Only collect the files required for the current platform, making each individual platform .zip smaller, and not requiring other platforms to be present.